org.eclipse.jdt.debug.tests/tests/org/eclipse/jdt/debug/tests/core/VMInstallTests.java

```diff
@@ -318,7 +320,10 @@ private AutoCloseableSupplier<IVMInstall> searchFirstNonModularVM() throws IOExc
 				vmType.disposeVMInstall(id);
 			}
 		}
-		assertNotNull("No non-modular VM (<=Java-1.8) found.", candidate);
+		if (candidate == null) {
```

Comment:
`&& System.getenv("NON_MODULAR_JAVA_HOME") == null`
?
This way we would see a fail in Jenkins in case we will miss Java 8 after IT changes

Reply:
Good idea, but I think the logic has to be slightly different.

Reply:
Ok, haven't seen it.